pkg/proxy/handlers/spacelister.go

```diff
@@ -36,9 +38,10 @@ func NewSpaceLister(app application.Application) *SpaceLister {
 }
 
 func (s *SpaceLister) HandleSpaceListRequest(ctx echo.Context) error {
-
+	startTime := ctx.Get(context.RequestRecievedTime).(time.Time)
 	workspaces, err := s.ListUserWorkspaces(ctx)
 	if err != nil {
+		metrics.RegServProxyResponseHistogramVec.WithLabelValues(metrics.ResponseMetricLabelRejected).Observe(time.Since(startTime).Seconds())
```

Comment:
Ok so as we discussed it was my mistake and the `ProxyResponse` is the name used for requests that are not routed to the member cluster. I wonder if there's a better name though since `ProxyResponse` sounds like it is focused on the response time but that's not the case right? Would `RegServProxyNotRoutedHistogramVec` be more clear? Open to other suggestions as well.

Reply:
@rajivnathan I understand the confusion, does `RegServProxyResolverHistogramVec` help? basically then we have two histogramVecs, one for routing and one for resolving.

Reply:
Hmm, Routing and Resolving still sound similar to me but that could be just me.

Reply:
we resolved the naming in sync call, and here are the new names [12580c1](https://github.com/codeready-toolchain/registration-service/pull/342/commits/12580c1cb0d601c673b1516f5938ea5ee078414c)